dev/PushNotifications/PushNotificationManager.h

```diff
@@ -4,6 +4,23 @@
 #pragma once
 #include "Microsoft.Windows.PushNotifications.PushNotificationManager.g.h"
 
+namespace winrt::Microsoft::Windows::PushNotifications::implementation
```

Comment:
Can you please move this back down to where the definition was earlier? It makes it easier to diff and compare the changes.